docs/components/content/examples/FormExampleJoi.vue

```diff
@@ -29,11 +29,11 @@ async function submit () {
     :state="state"
     @submit.prevent="submit"
   >
-    <UFormGroup label="Email" name="email-joi">
```

Comment:
Indeed, I see the issue. I've updated this because it throws some duplicate ids warnings in the console.

Reply:
Ok! Let's update the schema path then!

Reply:
I looked into it, is it necessary to bind the name props to id on inputs?

Reply:
Not really, it was just a shortcut to write less code.

Reply:
Oh sorry, the `id` is here to be targeted by the FormGroup component through `<label for="..." />`, we can't remove it..

Reply:
What can we do to fix this quickly? I plan to release the `2.7.0` today.

Reply:
I changed the name attributes in the documentation, should be enough for now

Reply:
> Oh sorry, the id is here to be targeted by the FormGroup component through <label for="..." />, we can't remove it..

Could you send me a link to the code?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes